cognite_toolkit/_builtin_modules/inrobot/cdf_inrobot_location/robotics/root.Map.yaml

```diff
@@ -1,10 +1,10 @@
-name: "{{ three_d_model_name }}"
-externalId: "{{ three_d_model_name }}"
-description: "{{ three_d_type }} {{ three_d_model_name }}"
-mapType: "{{ three_d_type }}"
-frameExternalId: "frame_{{ three_d_model_name }}"
+name: {{ three_d_model_name }}
+externalId: {{ three_d_model_name }}
+description: {{ three_d_type }} {{ three_d_model_name }}
+mapType: {{ three_d_type }}
+frameExternalId: frame_{{ three_d_model_name }}
 data:
-  modelId: "{{ three_d_model_id }}"
-  revisionId: "{{ three_d_revision_id }}"
+  modelId: {{ three_d_model_id }}
+  revisionId: {{ three_d_revision_id }}
```

Comment:
This is what was causing the issue. The `"` causes `modelId` and `revisionId` to be read as string and not int. When trying to deploy it, the API will fail.